incubator/helm-release-pruner/templates/rbac.yml

```diff
@@ -30,3 +30,25 @@ subjects:
 - kind: ServiceAccount
   name: {{ template "helm-release-pruner.fullname" . }}
   namespace: {{ .Release.Namespace }}
+---
```

Comment:
I'd like to avoid giving the service account access to all namespaces to avoid accidentally deleting staging or other non-ephemeral deploys. That seems like a bad default for the chart to grant.

I had put the role binding in the app deploy to accomplish this originally: https://github.com/trueandco/trueandco-web/blob/master/deploy/ephemeral/helm-release-pruner.service_account.yml

Now that rbac-manager supports label based bindings I'd lean towards removing all bindings here and in the app deploy and add a label based privilege for the pruner service account in the reckoner rbac definitions file.

What would you think about that?

Reply:
that sounds logical, as long as we're ok with having a dependency on a recent version of rbac-manager

Reply:
I added an RbacDefinition thats disabled by default, and removed the namespace access role.